annotationsx/settings/base.py

```diff
@@ -25,7 +25,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = literal_eval(os.environ.get('DEBUG', str(SECURE_SETTINGS.get('debug', True))))
 
-ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', SECURE_SETTINGS.get('allowed_hosts', []))
+ALLOWED_HOSTS = ['localhost',  '127.0.0.1']
+allowed_hosts_other = os.environ.get(
+    'ALLOWED_HOSTS', SECURE_SETTINGS.get('allowed_hosts', ''))
+if allowed_hosts_other:
+    ALLOWED_HOSTS.extend(allowed_hosts_other.split())
```

Comment:
What are you expecting allowed_hosts_other to be? I have this saved as an array in secure.py so it is hitting an error when you're trying to call the split function on a list.